test/ClangImporter/inlinable_bitfields.swift

```diff
@@ -11,4 +10,4 @@ public func g(_ m: MM) -> UInt32 {
 
 // Just make sure this is a definition and not a declaration...
 
-// CHECK: define internal i32 @"$ModRM$rm$getter"
+// CHECK: define internal zeroext i32 @"$ModRM$rm$getter"
```

Comment:
Won't this break platforms that aren't using `zeroext`?

You can use a FileCheck regex to handle this: `{{( zeroext)?}}`.

Reply:
Updated to use {{( zeroext)?}}. Pls review. Thanks!